### DISCUSSION TITLE
SheepRL's DreamerV3 plays Super Hang-On.

### DISCUSSION THREAD

**Discussion by Disastorm** [Eclectic-Sheep/sheeprl#299] [Show and tell]
I made a video of a single DreamerV3 model completing all four courses in Super Hang-On. 

https://www.youtube.com/watch?v=IxrNMrVxxCs

**Comment by Xritsos** [comment]
How many steps did it take you to learn? Also what about training times, because I have a simple experiment and it is very slow (10,000 steps at ~1.5 hours). Is this normal?

**Reply by Disastorm** [reply]
sorry don't remember, it was a long time ago. I think step counts for dreamer v3 are lower than something like PPO but each step takes longer. I think in general it ends up performing better than PPO if you train it for the same period of time.

**Reply by Xritsos** [reply]
Thanks for your reply. What do you actually mean by same period of time? Is it number of steps or clock time?

**Reply by Disastorm** [reply]
yea clock time. What I mean is from what I remember I think each step in dreamerv3 takes longer than a step in PPO. I know you didnt ask about PPO at all, but I think PPO is one of the most commonly used algorithms so thats just why I mentioned it.

And by performance for time I meant clock time, so I meant like with lower steps it would perform as good as or potentially better than PPO.
